core/src/types/transaction/mod.rs

```diff
@@ -198,398 +200,298 @@ pub struct InitValidator {
     pub validator_vp_code_hash: Hash,
 }
 
-/// Module that includes helper functions for classifying
-/// different types of transactions that the ledger
-/// must support as well as conversion functions
-/// between them.
-#[cfg(feature = "ferveo-tpke")]
-pub mod tx_types {
-    use std::convert::TryFrom;
+/// Struct that classifies that kind of Tx
+/// based on the contents of its data.
+#[allow(clippy::large_enum_variant)]
```

Comment:
I think we should box the `Wrapper` and `Protocol` variants instead otherwise 8 bytes in the header for the other variants get bloated to 560 bytes

Reply:
done in 0383703befc59fa1a67c621f3ae3a547a52277e5